spec/DatabaseSpec.js

```diff
@@ -62,18 +62,230 @@ describe('Database Spec', () => {
   });
   describe('Recommendations Table', () => {
     it('Should generate a recommendation', (done) => {
-      const obj = {
-        market: 'San Francisco',
-        roomType: 'Shared room',
-        checkIn: '2017-11-11',
-        checkOut: '2018-11-17',
-      };
-      generateRecommendation(obj)
-        .then((price) => {
-          expect(price.rows[0].avg).to.be.above(200);
+      const obj = `{
+        "topic": "search-availability",
+        "payload": {
+          "searchEventId": "gt9s1syij9em2uf5",
+          "searchRequest": {
+            "timestamp": "2017-10-30T20:02:49.600Z",
+            "visitId": "258",
+            "userId": "459023",
+            "market": "San Francisco",
+            "checkIn": "2017-11-12",
+            "checkOut": "2017-11-14",
+            "roomType": "any",
+            "limit": "10"
+          },
+          "searchResults": [
+            {
+              "listingId": 5858,
+              "listingName": "Creative Sanctuary",
+              "hostName": "Philip And Tania",
+              "market": "San Francisco",
+              "neighbourhood": "Bernal Heights",
+              "roomType": "Entire home\/apt",
+              "nightlyPrices": [
+                {
+                  "date": "2017-11-12T08:00:00.000Z",
+                  "price": "$235.00"
+                },
+                {
+                  "date": "2017-11-13T08:00:00.000Z",
+                  "price": "$235.00"
+                }
+              ],
+              "averageRating": 56,
+              "score": 0
+            },
+            {
+              "listingId": 7918,
+              "listingName": "A Friendly Room",
+              "hostName": "Aaron",
+              "market": "San Francisco",
+              "neighbourhood": "Haight Ashbury",
+              "roomType": "Private room",
+              "nightlyPrices": [
+                {
+                  "date": "2017-11-12T08:00:00.000Z",
+                  "price": "$63.00"
+                },
+                {
+                  "date": "2017-11-13T08:00:00.000Z",
+                  "price": "$63.00"
+                }
+              ],
+              "averageRating": 24,
+              "score": 1
+            },
+            {
+              "listingId": 8014,
+              "listingName": "Newly Remodeled room in big house WIFI market",
+              "hostName": "Jia",
```

Comment:
Are you generating listingNames for your database the same way you are generating them for this test spec?
 Might consider looking at the way listingNames are created

Reply:
Hey Ryan, this is Christine's data that I grabbed from her text file! If it's not in that form from your inventory, might want to check with @christine-w

Reply:
Tyler, I was referring to how the names were being generated, "Newly Remodeled room in big house WIFI market" looks a interesting